client/capability-token-2way/capability-token.5.x.rb

```diff
@@ -4,13 +4,13 @@
 get '/token' do
   account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
   auth_token = 'your_auth_token'
-  capability = Twilio::JWT::Capability.new(account_sid, auth_token)
+  capability = Twilio::JWT::ClientCapability.new(account_sid, auth_token)
```

Comment:
AccessToken is the preferred way, but if this fixes the brokenness then 👍

Reply:
Yup, mainly about being broken.